src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -91,6 +91,7 @@ class HoldRequest extends React.Component {
     let path = `${appConfig.baseUrl}/hold/confirmation/${bibId}-${itemId}`;
     const searchKeywordsQuery =
       (this.props.searchKeywords) ? `searchKeywords=${this.props.searchKeywords}` : '';
+    const searchKeywordsQueryPhysical = searchKeywordsQuery ? `&${searchKeywordsQuery}` : '';
 
     if (this.state.delivery === 'edd') {
       const searchKeywordsQueryEdd = searchKeywordsQuery ? `?${searchKeywordsQuery}` : '';
```

Comment:
`searchKeywordsQueryPhysical` and `searchKeywordsQueryEdd` are very similar except that one has `&` and the other `?`. Maybe update it so there's one variable and you have `&` and `?` in the routes below?